pontificia-universidade-catolica-do-parana-abnt.csl

```diff
@@ -1,216 +1,465 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="never" default-locale="pt-BR">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" initialize="false" demote-non-dropping-particle="never" default-locale="pt-BR">
   <info>
-    <title>Pontificia Universidade Catolica do Parana - ABNT (Portuguese - Brazil)</title>
-    <title-short>PUC-PR-ABNT</title-short>
+    <title>Pontificia Universidade Catolica do Paraná - (autoria completa) (Português - Brasil)</title>
+    <title-short>PUC-PR-ABNT-completa</title-short>
     <id>http://www.zotero.org/styles/pontificia-universidade-catolica-do-parana-abnt</id>
     <link href="http://www.zotero.org/styles/pontificia-universidade-catolica-do-parana-abnt" rel="self"/>
     <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas" rel="template"/>
-    <author>
-      <name>Pontificia Universidade Catolica do Parana</name>
-      <email>biblioteca@pucpr.br</email>
-      <uri>https://www.pucpr.br</uri>
-    </author>
+    <!-- Estilo original utilizado como modelo -->
+    <!-- Para autor ficar com nome completo, é necessário preencher com nome completo dos autores na biblioteca. Ver documentação sobre preenchimento dos campos básicos para ABNT -->
+    <!-- Para alguns itens do Mendeley é necessário incluir novos campos -->
+    <!-- Para alguns itens do Zotero é necessário utilizar o campo "extra". Exemplo: incluir "publisher-place:" para artigos de periódico.  -->
+    <contributor>
+      <name>Giovanna Carolina Massaneiro dos Santos</name>
+      <email>giovanna.massaneiro@pucpr.br</email>
+      <uri>https://www.pucpr.br/biblioteca/</uri>
+    </contributor>
     <category citation-format="author-date"/>
-    <summary>De acordo com ABNT-NBR 10520:2023 e ABNT-NBR 6023:2018</summary>
-    <updated>2025-11-10T17:27:05+00:00</updated>
+    <category field="generic-base"/>
+    <summary>The Brazilian Standard Style in accordance with ABNT-NBR 10520.2023 and ABNT-NBR 6023.2018</summary>
+    <updated>2025-11-25T14:45:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <locale xml:lang="pt">
+  <locale xml:lang="pt-BR">
     <terms>
-      <term name="director" form="verb">direção</term>
-      <term name="volume" form="short">v.</term>
+      <term name="accessed">acesso</term>
+      <term name="available at">disponível em</term>
```

Comment:
This term is already included in our [pt-BR locale](https://github.com/citation-style-language/locales/blob/master/locales-pt-BR.xml). Any reason why this was added here?